src/loadedPlugins.ts

```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated. DO NOT EDIT.
+
+import Plugin_github from '../plugins/github/index';
+import Plugin_myotherplugin from '../plugins/myotherplugin/index';
+import Plugin_myplugin from '../plugins/myplugin/index';
+
+import {PluginRegistry} from './types/plugin_registry';
+
+export function initPlugins(pluginRegistry: PluginRegistry) {
+  Plugin_github.init(pluginRegistry);
+  Plugin_myotherplugin.init(pluginRegistry);
+  Plugin_myplugin.init(pluginRegistry);
+}
```

Comment:
This file is automatically populated at compile-time using `generatePluginImports.js`. It scans the `plugins` folder and creates the imports in this file.